system-addon/content-src/activity-stream-windows.scss

```diff
@@ -0,0 +1,7 @@
+/* This is the windows variant */
+@import './activity-stream';
+
+// Search border should only have a 1-pixel blue border, so no box-shadow is needed.
+.search-wrapper input:focus {
+  box-shadow: none;
```

Comment:
The change from #3785 removes the border, so we'll have to fix this a different way.